### DISCUSSION TITLE
Skipping initialization hours?

### DISCUSSION THREAD

**Discussion by malloryprow** [dtcenter/METplus#1760] [Configuration] [answered]
Hi everyone,

In the METplus documentation, I see the configuration option "[SKIP_TIMES](https://metplus.readthedocs.io/en/main_v4.0/Users_Guide/systemconfiguration.html?highlight=SKIP_TIMES)". It sounds like this pertains only to valid times. Is there any way to skip initialization times?

I'm verifying the GFS against observations from the NAM prepbufr files at valid times 00, 03, 06, 09, 12, 15, 18, and 21Z. We want verification every 3 forecast hours. The GFS is only initialized at 00, 06, 12, and 18Z, so when the verification is running I get error messages that it is missing files for 03, 09, 15, and 21Z (which is expected of course). Since there are missing forecast files this causes METplus to exit with a non-zero status.

**Comment by georgemccabe** [comment, marked as answer]
Hi @malloryprow,

You are correct that the SKIP_TIMES logic only applies to valid times. [Looping by initialization time](https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#looping-by-initialization-time) (`LOOP_BY=INIT`) is the easiest way to ensure you are only processing the init times that are available.

If you need to be looping by valid time, then the [INIT_SEQ](https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#init-seq) may accomplish what you need. You would specify the list of the available initialization times along with the min/max forecast leads to use and the wrapper will determine which forecast leads can be used.

The following settings will process forecast leads between 3 and 12 that align with the current valid time and available init times:
```
INIT_SEQ = 0,6,12,18
LEAD_SEQ_MIN = 3
LEAD_SEQ_MAX = 12
```

For Valid 0Z:
* Forecast lead 6hr = init 18Z on previous day
* Forecast lead 12hr = init 12Z on previous day

For Valid 3Z:
* Forecast lead 3hr = init 0Z
* Forecast lead 9hr = init 18Z on previous day

...

For Valid 21Z:
* Forecast lead 3hr = init 18Z
* Forecast lead 9hr = init 12Z

**Reply by malloryprow** [reply]
Thanks George! I'll play around with that and see how it goes!

**Reply by malloryprow** [reply]
This worked great, George! Thanks!

**Reply by georgemccabe** [reply]
Glad to hear it! I am going to lock this discussion to prevent future posts. Please feel free to create a new discussion if you have any other questions or issues.
